views/js/e2e/runner/tools/tools.spec.js

```diff
@@ -39,46 +39,422 @@ describe('Tools', () => {
     before(() => {
         cy.setupServer();
         cy.addRoutes();
-        cy.login('admin');
-        cy.importTestPackage(base64Test, 'e2e Tools test');
-        cy.publishTest('e2e Tools test');
-        cy.setDeliveryForGuests('e2e Tools test');
-        cy.logout();
+        cy.addExecutionRoutes();
+        // cy.login('admin');
+        // cy.importTestPackage(base64Test, 'e2e Tools test');
+        // cy.publishTest('e2e Tools test');
+        // cy.setDeliveryForGuests('e2e Tools test');
+        // cy.logout();
+        cy.guestLogin();
+        cy.startTest('e2e Tools test');
     });
 
     /**
      * Log in & start the test
      */
     beforeEach(() => {
-        cy.guestLogin();
-        // cy.startTest('e2e Tools test'); // TODO:
+        cy.setupServer();
+        cy.addRoutes();
+        cy.addExecutionRoutes();
+        // cy.guestLogin();
+        // cy.startTest('e2e Tools test');
     });
 
     /**
      * Log out
      */
     afterEach(() => {
-        cy.guestLogout();
+        // cy.guestLogout();
     });
 
     /**
      * Destroy everything we created during setup, leaving the environment clean for next time.
      */
     after(() => {
-        cy.guestLogout();
-        cy.login('admin');
-        cy.deleteItem('e2e Tools test');
-        cy.deleteTest('e2e Tools test');
-        cy.deleteDelivery('Delivery of e2e Tools test');
+        // cy.login('admin');
+        // cy.deleteItem('e2e Tools test');
+        // cy.deleteTest('e2e Tools test');
+        // cy.deleteDelivery('Delivery of e2e Tools test');
     });
 
     /**
      * Tools tests
      */
     describe('Test-Taker Tools', () => {
 
-        it('List contains Tools e2e delivery', function() {
-            cy.get(runnerSelectors.testList).find(runnerSelectors.availableDeliveries).contains('Delivery of e2e Tools test');
+        it('Has comments tool', function() {
```

Comment:
We could separate such big blocks to multiple "it" sections

Reply:
It's done now.